lib/zlib/adler32.js

```diff
@@ -3,6 +3,25 @@
 // Note: adler32 takes 12% for level 0 and 2% for level 6.
 // It doesn't worth to make additional optimizationa as in original.
 // Small size is preferable.
+//
+// (C) 1995-2013 Jean-loup Gailly and Mark Adler
+// (C) 2014-2017 Vitaly Puzrin and Andrey Tupitsin
+//
+// This software is provided 'as-is', without any express or implied
```

Comment:
I have no principal objections, but is it necessary to add such big text to every header (from this line and below)? Looks boring. Even original zlib sources don't have such thing.